### PR TITLE
Cleanup usage message

### PR DESCRIPTION
**Pull request by gerardroche**

### CODE REVIEW

**Comment by rogeriopradoj** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by CHH** [comment]
👍🏻

File: bin/php-build [hunk 1]
```diff
@@ -17,33 +17,30 @@
...
+# The most important line in every shell script. Basically this says: This
+# script should fail once a command invocation itself fails.
 set -e
 
-# This is the usage message, like it's done in
-# [shocco](https://github.com/rtomayko/shocco).
-#
-#/ php-build, Compile and Install PHP
-#/ Version: {{VERSION}}
+#/ usage: php-build [options] <definition> <prefix>
```
Comment (CHH) [inline bin/php-build]: Why "usage" instead of "Usage"?

Comment (gerardroche) [inline bin/php-build]: Just a convention. I just checked a few commands to see what others do:

- `git` lowercase
- `tmux` lowercase
- `ls` uppercase
- `cp` uppercase

My personal preference is lowercase. I like the git coding conventions. https://github.com/git/git/blob/master/Documentation/CodingGuidelines

Comment (CHH) [inline bin/php-build]: :+1:

**Comment by CHH** [comment]
lgtm

**Comment by CHH** [comment]
Thanks!

**Pull request merged**